### PR TITLE
3 more template projects

### PR DESCRIPTION
**Pull request by milendyankov**
Hi Ray, here are 3 more projects I created recently. I also changed the README to be more about extension points we have, than about the actual template projects. Personally I think this way is more intuitive. Let me know what you think.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,228 +47,434 @@ Liferay does not provide a blueprint implementation out of the box. To use the b
 
 Simply download the bundles from mvnrepository and drop them in your osgi/modules folder as usual before deploying blueprint bundles.
 
-## Gradle
 
-* `/gradle/blade.authenticator.shiro`
 
-> A gradle project which uses Apache Shiro for hooking auth.pipeline.pre.
+# Liferay extension points and template projects
```
Comment (gamerson) [inline README.md]: What @milendyankov  what an improvement!  looks so much better now.

**Comment by gamerson** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by AndyWu2015** [comment]
Hi @milendyankov  , did you test these three bundles in runtime ? I deployed them and model listener worked well but 
friendlyurl got following error:
<b> java.io.IOException: Unable to open resource in class loader META-INF/friendly-url-routes/routes.xml</b>
and 
rest got :
<b>Unresolved requirement: Import-Package: javax.ws.rs; version="[2.0.0,3.0.0)" in osgi </b>

**Comment by milendyankov** [comment]
Hi @AndyWu2015 , of course I did! 
This is how I discovered LPS-61293 and LPS-60997 :)
So once https://github.com/rotty3000/liferay-portal/pull/1085 and https://github.com/rotty3000/liferay-portal/pull/1087 are merged it will all work fine :)

**Comment by gamerson** [comment]
Hey Andy did you follow the setup guide that Milen put in the readme?
There is some extra setup you have to do.  However, the "unable to open
resource in class loader" doesn't look too friendly.

On Tue, Dec 15, 2015 at 2:39 AM, Andy Wu <notifications@github.com> wrote:

> Hi @milendyankov <https://github.com/milendyankov> , did you test these
> three bundles in runtime ? I deployed them and model listener worked well
> but
> friendlyurl got following error:
> * java.io.IOException: Unable to open resource in class loader
> META-INF/friendly-url-routes/routes.xml*
> and
> rest got :
> *Unresolved requirement: Import-Package: javax.ws.rs <http://javax.ws.rs>;
> version="[2.0.0,3.0.0)" in osgi *
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/rotty3000/blade/pull/186#issuecomment-164685924>.
>



-- 
Greg Amerson
Liferay Developer Tools
Liferay, Inc. www.liferay.com

**Pull request merged**